ci-operator/step-registry/openshift/microshift/infra/rpm-install-aws/openshift-microshift-infra-rpm-install-aws-commands.sh

```diff
@@ -17,7 +17,11 @@ EOF
 
 configure_vm_args=""
 if "${OPTIONAL_RPMS}"; then
-  configure_vm_args="--optional-rpms"
+  configure_vm_args="--skip-optional-rpms"
+fi
+
+if "${SKIPPED_OPTIONAL_RPMS}"; then
```

Comment:
This block should probably go under `if ${OPTIONAL_RPMS}`?

```suggestion
if [ -n "${SKIPPED_OPTIONAL_RPMS}" ] ; then
```